tensorhive/config.py

```diff
@@ -97,7 +97,7 @@ def hosts_config_to_dict(path: str) -> Dict:
             if section == 'proxy_tunneling':
                 continue
 
-            # TODO Handle more options (https://github.com/ParallelSSH/parallel-ssh/blob/2e9668cf4b58b38316b1d515810d7e6c595c76f3/pssh/clients/base_pssh.py#L119)
+            # TODO Handle more options (https://github.com/ParallelSSH/parallel-ssh/blob/2e9668cf4b58b38316b1d515810d7e6c595c76f3/pssh/clients/base_pssh.py#L119) # NOQA: E501
```

Comment:
This entire comment can be removed - deprecated.

Reply:
fixed in the following commit